src/main/resources/config/application-prod.yml

```diff
@@ -102,21 +114,22 @@ jhipster:
             timeToLiveInDays: 1461
     security:
         client-authorization:
-            client-id:
-            client-secret:
+            client-id: internal
+            client-secret: internal
+        authentication:
+            jwt:
+                # Access Token is valid for 5 mins
+                token-validity-in-seconds: 300
+                # Refresh Token is valid for 7 days
+                token-validity-in-seconds-for-remember-me: 252000
     mail: # specific JHipster mail property, for standard properties see MailProperties
-        from: <tenantname>@xm-online.com
+        from: uaa@localhost
```

Comment:
As far as i know there is some dependency in mail sending code which should use placeholder. So please check if it is not break the functionality.

Reply:
fixed